dayrepo/snippets/views.py

```diff
@@ -193,32 +196,244 @@ def post(self, request):
+            sheet["X27"] = process_date[i].client
+            sheet["AJ27"] = process_date[i].goods
+            sheet["AS27"] = process_date[i].load_situation
+            if process_date[i].is_load_situation != False:
+                sheet["BD27"] = "良"
+            if process_date[i].load_mileage != False:
+                sheet["BG27"] = process_date[i].load_mileage
+            if process_date[i].hollow_mileage != False:
+                sheet["BI27"] = process_date[i].hollow_mileage
+        if i == 3:
+            sheet["C29"] = process_date[i].start_point
+            sheet["G29"] = process_date[i].via_point
+            sheet["O29"] = process_date[i].end_point
+            sheet["X29"] = process_date[i].client
+            sheet["AJ29"] = process_date[i].goods
+            sheet["AS29"] = process_date[i].load_situation
+            if process_date[i].is_load_situation != False:
+                sheet["BD29"] = "良"
+            if process_date[i].load_mileage != False:
+                sheet["BG29"] = process_date[i].load_mileage
+            if process_date[i].hollow_mileage != False:
+                sheet["BI29"] = process_date[i].hollow_mileage
+        if i == 4:
+            sheet["C31"] = process_date[i].start_point
+            sheet["G31"] = process_date[i].via_point
+            sheet["O31"] = process_date[i].end_point
+            sheet["X31"] = process_date[i].client
+            sheet["AJ31"] = process_date[i].goods
+            sheet["AS31"] = process_date[i].load_situation
+            if process_date[i].is_load_situation != False:
+                sheet["BD31"] = "良"
+            if process_date[i].load_mileage != False:
+                sheet["BG31"] = process_date[i].load_mileage
+            if process_date[i].hollow_mileage != False:
+                sheet["BI31"] = process_date[i].hollow_mileage
+        if i == 5:
+            sheet["C33"] = process_date[i].start_point
+            sheet["G33"] = process_date[i].via_point
+            sheet["O33"] = process_date[i].end_point
+            sheet["X33"] = process_date[i].client
+            sheet["AJ33"] = process_date[i].goods
+            sheet["AS33"] = process_date[i].load_situation
+            if process_date[i].is_load_situation != False:
+                sheet["BD33"] = "良"
+            if process_date[i].load_mileage != False:
+                sheet["BG33"] = process_date[i].load_mileage
+            if process_date[i].hollow_mileage != False:
+                sheet["BI33"] = process_date[i].hollow_mileage
+
+    sheet["AO8"] = (
+        snippet_date.checklist_id.account_id.last_name
+        + " "
+        + snippet_date.checklist_id.account_id.first_name
+    )
     # Excelを返すためにcontent_typeに「application/vnd.ms-excel」をセットします。
-    response = HttpResponse(content_type='application/vnd.ms-excel')
-    response['Content-Disposition'] = 'attachment; filename = report.xlsx'
+    response = HttpResponse(content_type="application/vnd.ms-excel")
```

Comment:
結局 application/vnd.ms-excel の指定だと xls を指定するみたいだが、あえて古い方を指定しているのには何か意図がある? 
https://developer.mozilla.org/ja/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

なければ、使用しているライブラリ的にもこっちだと思う。
```suggestion
    response = HttpResponse(content_type="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet")
```

Reply:
動作確認して問題なければ変更します